packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/add/dedicatedCloud-datacenter-add.controller.js

```diff
@@ -19,23 +26,75 @@ export default class {
     };
 
     this.load();
-
-    this.DedicatedCloud.getOptionState('nsx', this.serviceName).then((data) => {
-      this.nsxStatus = data;
-    });
   }
 
   load() {
     this.loader = true;
     this.DedicatedCloud.getCommercialRangeCompliance(this.serviceName)
       .then((compliance) => {
         this.commercialRange.list = compliance;
+        this.calculateEstimate();
```

Comment:
Your finally block needs to me moved upper

```suggestion
       return this.calculateEstimate();
```
And then you can add your block finally
```
      .catch((err) => {
        this.goBack(
          `${this.$translate.instant(
            'dedicatedCloud_datacenters_adding_load_error',
          )} ${err.message || ''}`,
          'danger',
        );
      })
      .finally(...);

Reply:
Exactly, i fixed.